var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -132,13 +166,22 @@ def setup_build_environment(self, env):
         env.set("BAZEL_LINKOPTS", "")
         env.set("BAZEL_LINKLIBS", "-lstdc++")
 
-        env.set(
-            "EXTRA_BAZEL_ARGS",
-            # Spack's logs don't handle colored output well
-            "--color=no --host_javabase=@local_jdk//:jdk"
-            # Enable verbose output for failures
-            " --verbose_failures --jobs={0}".format(make_jobs),
+        # .WARNING: Option 'host_javabase' is deprecated
+        # Use local java installation
+        args = "--color=no --define=ABSOLUTE_JAVABASE={0} --verbose_failures --jobs={1}".format(
+            self.spec["java"].prefix, make_jobs
         )
+        for resource_name in self.resource_dictionary.keys():
+            if resource_name:
+                if self.spec.satisfies(self.resource_dictionary[resource_name]["when"]):
+                    archive_path = self.stage.source_path
+                    archive_path = archive_path.replace(
+                        "spack-stage-bazel-{0}".format(self.version),
+                        "resource-{0}".format(resource_name),
+                    )
```

Comment:
@scheibelp @becker33 anyone know a better way to get the path to a resource?

Reply:
Since we're in an instance method, these can be accessed like `Package.stage[1:]`:

```
resource_stages = self.stage[1:]
for x in resource_stages:
  the_dir_with_the_downloaded_artifact = x.source_path
  ...
```

Reply:
Thank you, I will use it.